lnrpc/rpc.proto

```diff
@@ -1660,13 +1660,15 @@ message ChannelEventUpdate {
         ChannelCloseSummary closed_channel = 2 [ json_name = "closed_channel" ];
         ChannelPoint active_channel = 3 [ json_name = "active_channel" ];
         ChannelPoint inactive_channel = 4 [ json_name = "inactive_channel" ];
+        ChannelOpenUpdate pending_open_channel = 6 [json_name = "pending_open_channel"];
```

Comment:
The reason why I used `ChannelOpenUpdate` is because it is the same type that is used for the `OpenChannel` stream. Both use the data and gets triggered in the same place (https://github.com/lightningnetwork/lnd/pull/3582/files#diff-691ca64693e704fd2e5080998f3e640eR1785-R1800).
I skipped `lnrpc.OpenStatusUpdate` though.

If we want to send more data from the `SubscribeChannelEvents` stream in the future, I think it would be a good idea to use `ChannelOpenUpdate` instead of `ChannelPoint`.

If you think it makes more sense to `ChannelPoint` instead, I could change to that.

Reply:
Doesnt the stream use `PendingUpdate` here?

Reply:
@halseth You're right, `PendingUpdate` is what `OpenChannel` uses in this case.

Reply:
Fixed.